From: Jonathan Bergh <bergh.jonathan@gmail.com>
Date: Sat, 30 Sep 2023 11:04:21 +0200
Subject: ACPI: OSL: Remove redundant parentheses in return statements
Patch-mainline: v6.7-rc1
Git-commit: 9d4e27dd88275a8e2a14521b63792f33edca5758
References: jsc#PED-11164

Fix up three return statements including redundant perens around the
return value.

Signed-off-by: Jonathan Bergh <bergh.jonathan@gmail.com>
[ rjw: Subject and changelog edits ]
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Chun-Yi Lee <jlee@suse.com>
---
 drivers/acpi/osl.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/acpi/osl.c
+++ b/drivers/acpi/osl.c
@@ -1578,7 +1578,7 @@ acpi_os_create_cache(char *name, u16 siz
 acpi_status acpi_os_purge_cache(acpi_cache_t *cache)
 {
 	kmem_cache_shrink(cache);
-	return (AE_OK);
+	return AE_OK;
 }
 
 /*******************************************************************************
@@ -1597,7 +1597,7 @@ acpi_status acpi_os_purge_cache(acpi_cac
 acpi_status acpi_os_delete_cache(acpi_cache_t *cache)
 {
 	kmem_cache_destroy(cache);
-	return (AE_OK);
+	return AE_OK;
 }
 
 /*******************************************************************************
@@ -1617,7 +1617,7 @@ acpi_status acpi_os_delete_cache(acpi_ca
 acpi_status acpi_os_release_object(acpi_cache_t *cache, void *object)
 {
 	kmem_cache_free(cache, object);
-	return (AE_OK);
+	return AE_OK;
 }
 #endif
 
